osafw-app/App_Code/fw/FwModel.cs

```diff
@@ -332,7 +332,7 @@ public virtual int add(Hashtable item)
 
         this.removeCache(id);
 
-        if (!string.IsNullOrEmpty(field_prio))
+        if (!string.IsNullOrEmpty(field_prio) && string.IsNullOrEmpty(Utils.f2str(item[field_prio])))
```

Comment:
I would better check as `!item.ContainsKey(field_prio)`

Reply:
```prio``` field may be passed but empty. Probably:

` if (!string.IsNullOrEmpty(field_prio) && !(item.ContainsKey(field_prio) && !string.IsNullOrEmpty((string)item[field_prio]))`

Reply:
I would not check for the value itself, only for the presence of the key.
Reasoning - check for value is validation and on this level, there should be no validation, we can only add some default values if those are not passed.
Passing meaningful value is the responsibility of upper levels.

btw, a similar check is done a couple of lines above - `!item.ContainsKey(field_add_users_id)`

Reply:
okay, pull request updated